inst/cpp/sim.cpp

```diff
@@ -106,6 +106,7 @@ List sim_wrapper_cpp (NumericVector A, List design, List par, NumericVector iov_
       t_prv_dose = times[i];
       prv_dose = doses[i];
     }
+    // insert time-dependent covariates scale
```

Comment:
this is the label that `compile_sim_cpp` is looking for to replace with code. This needed to be moved up, to be executed before the call to `pk_code()`